crates/brainfuck_vm/src/bin/brainfuck_vm.rs

```diff
@@ -36,8 +36,10 @@ fn main() {
     let stdin = stdin();
     let stdout = stdout();
     let mut bf_vm = match args.ram_size {
-        Some(size) => Machine::new_with_config(&ins, stdin, stdout, size),
-        None => Machine::new(&ins, stdin, stdout),
+        Some(size) => {
+            Machine::new_with_config(&ins, stdin, stdout, size).expect("Failed to create machine")
+        }
+        None => Machine::new(&ins, stdin, stdout).expect("Failed to create machine"),
```

Comment:
Is it possible to avoid having to chain `.expect("")` everytime we want to instantiate a new Machine?

Reply:
- In main function yes, just fixed by returning `Result<(), MachineError>` for the function so that we can use the `?`
- Otherwise in tests this is common to have an `.expect("")` because due to the fact that we are providing a mocked setup, we are sure that the function will succeed (if not the test will panic and this is normal)

Reply:
Okay I see, thx